docs/tutorials/eda/eda-auto-target-analysis.md

```diff
@@ -0,0 +1,68 @@
+# EDA: Automated Target Variable Analysis
+
+In this section we explore automated dataset overview functionality.
+
+Automated target variable analysis aims to automatically analyze and summarize the variable we are trying to predict (
+label). The goal of this analysis is to take a deeper look into target variable structure and its relationship with
+other important variables in the dataset.
+
+To simplify outliers and useful patterns discovery. This functionality introducing components which allow generating
+descriptive statistics, visualizing the target distribution and relationships between the target variable and other
```

Comment:
Either you should replace the "," with an "and" since there are technically only two items you are listing ("generating ... and visualizing ...") or put a verb in front of "relationships" e.g. (generating ..., visualizing ... and describing relationships...)

Reply:
replaced with and